app/src/main/java/ch/epfl/culturequest/social/notifications/ScanNotification.java

```diff
@@ -10,14 +13,32 @@
 /**
  * Class that represents notifications for a new offline scan
  */
-public class ScanNotification implements Notification {
+public class ScanNotification implements NotificationInterface {
+    public static final String CHANNEL_ID = "SCAN";
 
     @Override
-    public android.app.Notification get(Context context) {
-        return new NotificationCompat.Builder(context, context.getString(R.string.scanNotifChannelID))
+    public Notification get(Context context) {
+        return new NotificationCompat.Builder(context, CHANNEL_ID)
                 .setSmallIcon(R.drawable.logo_compact)
-                .setContentTitle(Profile.getActiveProfile().getName() + ", you have a new scan!")
+                .setContentTitle(Profile.getActiveProfile().getUsername() + ", you have a new scan!")
                 .setContentText("We found a new offline scan result!")
                 .setPriority(NotificationCompat.PRIORITY_DEFAULT).build();
```

Comment:
This is only for offline scans right?

Reply:
Yes I assume so. Ahmed is the one who wrote it but I think it was meant for that.